app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ApplicationRecord
   has_secure_password
+  has_many :orders
```

Comment:
Again weirdness, We changed this to have the relationship and the password code Josh added. Gonna watch this when it's merged.